### DISCUSSION TITLE
Github action to upload BoMs to Dependency Track server

### DISCUSSION THREAD

**Discussion by antonroman** [DependencyTrack/dependency-track#1063] [Show and tell]
Just to let you know that we have just published a Github action to generate and upload BoM of Github repositories to a Dependency Track server.
 Currently only Python, Go and Nodejs projects are supported : [owasp-dependency-track-check](https://github.com/marketplace/actions/owasp-dependency-track-check)

Any feedback/contribution is really welcome.
Thanks in advance!

**Comment by stevespringett** [comment]
Fantasic. Your welcome to add the tool to https://github.com/DependencyTrack/dependency-track/blob/master/docs/_docs/integrations/community-integrations.md. Once 4.3 is released, the documentation will be updated and will include the tool.

**Reply by antonroman** [reply]
Done: https://github.com/DependencyTrack/dependency-track/pull/1066

Thanks :-)
